### DISCUSSION TITLE
Could you add support for AnimateLCM, please?

### DISCUSSION THREAD

**Discussion by JohnClaw** [leejet/stable-diffusion.cpp#169] [Ideas]
AnimateLCM provides fast video generation in 4 steps:
https://huggingface.co/wangfuyun/AnimateLCM

**Comment by Green-Sky** [comment]
![image](https://github.com/leejet/stable-diffusion.cpp/assets/2938071/a7ae1567-94ec-479f-b7b8-875491c556f1)

afaik only (1) is implemented rn.

**Comment by Amin456789** [comment]
nice find johnclaw, with svd img2vid and this one txt2vid sd cpp will be a full package
